### ISSUE TITLE
WebSocket support

### ISSUE THREAD

**Issue by MitchK** [mitre/HTTP-Proxy-Servlet#88]
Hi! 

Does this library support WebSockets and if not, is this planned?

Cheers

**Comment by dsmiley** [comment]
I don't think so.  I have no plans... I'm mostly a steward of pull requests these days.
